dahu/core/app/scripts/views/workspace/objects/tooltip.js

```diff
@@ -34,6 +35,10 @@ define([
 
         className: 'object tooltip ui-widget-content',
 
+        initialize: function() {
+            this.changeBackgroundColor();
```

Comment:
The default color is in the css file main.scss .tooltip. So when someone will change the default color he will have to change it in the css too. There is only {{{text}}} in the template of tooltips.